sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -195,7 +195,7 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
 
 object PROCESS_TABLES extends QueryTest with SQLTestUtils {
   // Tests the latest version of every release line.
-  val testingVersions = Seq("2.0.2", "2.1.2", "2.2.0", "2.2.1", "2.3.0")
```

Comment:
Add 2.3.1? (Note to self: add note in release guide to update this test.)

Reply:
yes, thanks! What about replacing "2.3.0" with "2.3.1"?

Reply:
I think we should follow the existing pattern; but let's do that in a separate change.

Reply:
ok, thanks, once this is merged I'll submit a new PR replacing "2.3.0" with "2.3.1" then, thanks.